jobs-builder/jobs/dependencies.yaml

```diff
@@ -0,0 +1,66 @@
+# This file contain the configurations to generate Kernel jobs.
+---
+- job-template:
+    name: 2.x-kernel-{flavor}-{arch}
+
+    # Template default variables
+    #
+    arch: x86_64
```

Comment:
Could this be specified as followsI wonder to avoid hard-coding the architecture (particularly since ARM is mentioned in the `.yaml.inc` file below):

```yaml
 arch: {arch}
```